crypto/fipsmodule/evp/p_pqdsa.c

```diff
@@ -261,27 +260,30 @@ EVP_PKEY *EVP_PKEY_pqdsa_new_raw_private_key(int nid, const uint8_t *in, size_t
   return NULL;
 }
 
-DEFINE_METHOD_FUNCTION(EVP_PKEY_METHOD, EVP_PKEY_pqdsa_pkey_meth) {
-  out->pkey_id = EVP_PKEY_PQDSA;
-  out->init = pkey_pqdsa_init;
-  out->copy = NULL;
-  out->cleanup = pkey_pqdsa_cleanup;
-  out->keygen = pkey_pqdsa_keygen;
-  out->sign_init = NULL;
-  out->sign = NULL;
-  out->sign_message = pkey_pqdsa_sign_message;
-  out->verify_init = NULL;
-  out->verify = NULL;
-  out->verify_message = pkey_pqdsa_verify_signature;
-  out->verify_recover = NULL;
-  out->encrypt = NULL;
-  out->decrypt = NULL;
-  out->derive = NULL;
-  out->paramgen = NULL;
-  out->ctrl = NULL;
-  out->ctrl_str = NULL;
-  out->keygen_deterministic = NULL;
-  out->encapsulate_deterministic = NULL;
-  out->encapsulate = NULL;
-  out->decapsulate = NULL;
-}
+const EVP_PKEY_METHOD pqdsa_pkey_meth = {
```

Comment:
Couldn't we continue to use `DEFINE_METHOD_FUNCTION` even thought it's not required outside of the FIPS module?

Reply:
For this I'd need to include delocator.h -- I did this before and Torben pointed out that there was a complaint about using the delocator out of the module. If my understanding is incorrect, I can modify.

Reply:
Ahhh. I forgot where this macro was defined.